sketch.js

```diff
@@ -39,17 +41,63 @@ function buttonsConfig(){
   buttons = {
     "add": createButton('Add!'),
     "label": createButton('Click me to change label!'),
-    "train": createButton('Train!')
+    "train": createButton('Train!'),
+    "auto_train": createButton('Auto train!')
+    // Added Button to start automatic training 
   }
   
-  buttons.add.position(50, 0);
-  buttons.train.position(0, 0);
-  buttons.label.position(94, 0);
+  buttons.add.position(50, 270);
+  buttons.train.position(0, 270);
+  buttons.label.position(94, 270);
+  buttons.auto_train.position(0, 290);
```

Comment:
Two ideas:
1. Change the hard-coded y-coordinate to a relative coordinate e.g. "canvas_width + 20"
2. Remove the label button in favor of a HTML drop-down menu (p5 does not offer these).